lib/openapi_parser/schemas/security_schemes.rb

```diff
@@ -0,0 +1,20 @@
+require 'jwt'
+module OpenAPIParser::Schemas
+  class SecuritySchemes < Base
+
+    openapi_attr_values :type, :description, :scheme
+    openapi_attr_value :bearer_format, schema_key: :bearerFormat
+
+    def validate_security_schemes(securityScheme, headers)
+      if self.type == "http" && self.scheme == "bearer" && self.bearer_format == "JWT"
+        raise "need authorization" unless headers["AUTHORIZATION"]
+        raise "not bearer" unless headers["AUTHORIZATION"].split[0] == "Bearer"
+
+        # check if the JWT token is being sent and try to decode.
+        # if JWT token does not exist or token cannot decode, then deny access
+        token = headers["AUTHORIZATION"].split[1]
+        value = JWT.decode token, nil, false
```

Comment:
How much detail to verify here is a difficult question.
When verifying a signed JWT, the user expects the signature to be correct.
But we must passed [key and various options](https://github.com/jwt/ruby-jwt) to the library.
The options are numerous and require the same amount of effort as creating a wrapper for the jwt library.
Also, the OpenAPI definition does not expect JWT to have detailed information about signature verification.

One solution would be to have the user pass in a callback what validation is to be performed, and this library would only call the callback.
This would allow users to, for example, check for certain values in the JWT or force disabling of the JWT for some users only.

In this case, instead of limiting the bearerFormat=JWT, I think it would be possible to pass a Security Scheme Object to the callback so that users can validate non-JWT formats as well, thus allowing for a wider range of applications.